pkg/tests/apis/dashboard/dashboards_test.go

```diff
@@ -31,8 +32,9 @@ func TestIntegrationRequiresDevMode(t *testing.T) {
 		t.Skip("skipping integration test")
 	}
 	helper := apis.NewK8sTestHelper(t, testinfra.GrafanaOpts{
-		AppModeProduction: true, // should fail
-		DisableAnonymous:  true,
+		AppModeProduction:    true, // should fail
+		DisableAnonymous:     true,
+		APIServerStorageType: options.StorageTypeUnifiedGrpc, // tests remote connection
```

Comment:
^^^ this runs all requests over a real grpc server on localhost -- the the inproc version